views/index.erb

```diff
@@ -1,9 +1,9 @@
-<h1>All My Students</h1>
+<h1>All My Favorite Bands</h1>
 <ul>
-  <% students.each do |student| %>
+  <% bands.each do |band| %>
     <li>
-      <a href="mailto:<%= student[:email] %>"><%= student[:name] %></a>
-      (<%= student[:discord] %>)
+      <a href="mailto:<%= band[:firstalbumyear] %>"><%= student[:name] %></a>
```

Comment:
An album year is not an email address; you cannot send an email to `"1968"` 😄 

The whole `<a href="mailto:blah">` format probably makes no sense for bands, and that's ok. Instead of just a simple `u`nordered `l`ist, you might want to use a `<table>` for your bands.